distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -577,6 +585,25 @@ public void testCswXmlIngest() {
     }
 
     @Test
+    public void testCswXmlWithHeaderIngest() {
```

Comment:
Actually on second thought I feel a little weird adding an integration test just to verify a test util 🤔 Besides that, does it add any coverage in the production code?

Really we should be unit testing test utils but I don't think that's being done anywhere yet